src/commands/ext-uninstall.ts

```diff
@@ -29,72 +30,82 @@ export default new Command("ext:uninstall <extensionInstanceId>")
       }
       throw err;
     }
-    let confirmedServiceAccountDeletion;
     if (!options.force) {
+      const serviceAccountMessage = `Uninstalling deletes the service account used by this extension instance:\n${clc.bold(
```

Comment:
Is it possible to add a line break between this first item of "Uninstalling does X" and the "This is what happens" line?

Reply:
Yep, will do!